tools/perl-version-convert

```diff
@@ -81,6 +81,8 @@ function convert_version
 	[[ "$DIST" == "Mail-AuthenticationResults" && "${VERSION:0:2}" == "2." ]] && ((${#VERSION} == 12)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VER[4]=${VERSION:11:1} && VERSION=${VERSION:0:1}
 	[[ "$DIST" == "Mail-DKIM" && "${VERSION:0:2}" == "1." ]] && ((${#VERSION} == 10)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VERSION=${VERSION:0:1}
 	[[ "$DIST" == "Mail-DKIM" && "${VERSION:0:2}" == "1." ]] && ((${#VERSION} == 12)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VER[4]=${VERSION:11:1} && VERSION=${VERSION:0:1}
+	[[ "$DIST" == "Mail-DMARC" && "${VERSION:0:2}" == "1." ]] && ((${#VERSION} == 10)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VERSION=${VERSION:0:1}
+	[[ "$DIST" == "Mail-DMARC" && "${VERSION:0:2}" == "1." ]] && ((${#VERSION} == 12)) && VER[1]=$((1${VERSION:2:4} - 10000)) && VER[2]=$((1${VERSION:6:2} - 100)) && VER[3]=$((1${VERSION:8:2} - 100)) && VER[4]=${VERSION:11:1} && VERSION=${VERSION:0:1}
```

Comment:
Why is this line needed?

Reply:
again Mail::DKIM had a version *.1 which would be covered by this second line. And Mail::DKIM/DMARC are coming from the same author just to avoid potentially extension.

Reply:
You cannot cover all possible future cases where there is no evidence they will happen and where their probability is very low.  `1.20200513.1` was real case for Mail::DKIM, but we have no such case for Mail::DMARC.

If you would want to cover such cases, then you should also cover a case when you'll have something like `1.20200513.9` and then `1.20200513.10` and then `1.20200513.11`.  These cases are uncovered.  Why you do not cover them?

Once we have a `1.20200513.1` case for Mail::DMARC we could easily start to handle it.